### ISSUE TITLE
Failed calibration - frequency not increasing each step

### ISSUE THREAD

**Issue by Sharkooon42** [bigtreetech/Eddy#103]
#44 
I'm starting this topic because at least three people, including myself, are still experiencing the same issue with Eddy USB calibration.Me with Eddy DUO. The error keeps occurring, and despite numerous discussions, there is no working solution or clear guidance on how to resolve the problem.

The main issue seems to be that the Z-axis loses steps or doesn't move correctly during the calibration process. However, it works perfectly fine during normal operations, which suggests that the problem is specifically related to the calibration routine.

It would be great if we could work together to find a solution or if anyone who has successfully resolved this issue could share their experience. Thank you!

**Comment by kasunami** [comment]
I ran into this issue after swapping my bed to a thicker plate. Unfortunately, I'm using a stock Ender 5 Pro Z-axis motion system, so the whole bed hangs off the back edge causing the whole thing to tilt forward like crazy. 

After a little digging, and just paying more attention to my printer... I found that the first few movements to sense the frequency changes actually had my nozzle tapping the bed. Digging through the Klipper code and then just paying attention to the message, the answer seemed clear: At least a couple of points got the same frequency reading, causing the error, likely due to the distance being the same for those movements where the nozzle tapped the bed.

TLDR; 
Make sure your Z axis is actually moving, no bed impacts, no lost steps. If you see any issues where the bed doesn't physically move away from the probe, that's the first thing to fix!
